spec.html

```diff
@@ -5219,6 +5219,17 @@
             </tr>
             <tr>
               <td>
+                [[GlobalThisValue]]
+              </td>
+              <td>
+                Object
+              </td>
+              <td>
+                The value returned by `this` in global scope. This may be the same as the global object, or may not. For example, in HTML the `this` value is a `WindowProxy` object, whereas the global object is a `Window`.
```

Comment:
I don't think this example is required or illustrative for ECMAScript spec readers. I would say something like:

> The value returned by `this` in global scope. This is usually the same object as the global object, however hosts may provide any ECMAScript Object value.

Has the added benefit of making it a normative requirement that the this value be an object, which I think is good :)